package/tests/test_views.py

```diff
@@ -114,20 +114,25 @@ def test_add_example_view(self):
 
         # Once we log in the user, we should get back the appropriate response.
         self.assertTrue(self.client.login(username='user', password='user'))
+        user = User.objects.get(username='user')
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'package/add_example.html')
 
-        count = PackageExample.objects.count()
+        id_list = list(PackageExample.objects.values_list('id', flat=True))
         response = self.client.post(url, {
             'title': 'TEST TITLE',
             'url': 'https://github.com',
         })
         self.assertEqual(response.status_code, 302)
-        self.assertEqual(PackageExample.objects.count(), count + 1)
+        self.assertEqual(PackageExample.objects.count(), len(id_list) + 1)
```

Comment:
This might be failing because in Django 1.8.x in some cases we're having to clean out table stores with things like `PackageExample.objects.all().delete()`